conseil-common/src/main/scala/tech/cryptonomic/conseil/common/config/Platforms.scala

```diff
@@ -125,6 +125,7 @@ object Platforms {
       network: String,
       enabled: Boolean,
       node: TezosNodeConfiguration,
+      bakerRollsSize: BigDecimal,
```

Comment:
why is it big decimal? shouldn't `Int` be enough?

Reply:
originally it was `BigDecimal`

Reply:
The entire `TezosType` domain is built around `PositiveBigNumber` type being a wrapper for BigDecimal. Therefore, the remaining functionality expects `BigDecimal` as well.